dlrs/clearlinux/releasenote.md

```diff
@@ -0,0 +1,136 @@
+
+# The Deep Learning Reference Stack Release
+
+To offer more flexibility, we are releasing multiple versions of the Deep Learning Reference Stack. All versions are built on top of the Clear Linux OS, which is optimized for IA.
+
+> **Note:**
+     The minimum validated version of Clear Linux for this stack is 31290.
+
+## The Deep Learning Reference Stack with Intel® MKL-DNN and Intel® AVX512-Deep Learning Boost
+
+The release includes:
+  * Clear Linux* OS
+  * Runtimes (python)
+  * TensorFlow 2.0 optimized using Intel® Math Kernel Library for Deep Neural Networks (Intel® MKL-DNN) primitives and Intel® AVX-512 Deep Learning Boost (Formerly Intel® VNNI)
+  * Intel OpenVINO model server v2019_R3
+
+## The Deep Learning Reference Stack with Intel® MKL-DNN and Intel® AVX512-Deep Learning Boost
+
+The release includes:
+  * Clear Linux* OS
+  * Runtimes (python)
+  * TensorFlow 1.15 optimized using Intel® Math Kernel Library for Deep Neural Networks (Intel® MKL-DNN) primitives and Intel® AVX-512 Deep Learning Boost (Formerly Intel® VNNI)
+  * Intel OpenVINO model server v2019_R3
+
+## The Deep Learning Reference Stack with Eigen
+
+The release includes:
+  * Clear Linux* OS
+  * TensorFlow 1.14 compiled with AVX2 and AVX512 optimizations
+  * Runtimes (python)
+
+> **Note:**
+   When using the Deep Learning Reference Stack with Intel®  MKL-DNN version, you may see this warning message: "tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 AVX512F FMA". This is because this version of the Deep Learning Reference Stack is using Intel®  MKL-DNN for performance optimization rather than Intel®  Advanced Vector Extensions 512, and is expected for this version of the Deep Learning Reference Stack.
+
+## The Deep Learning Reference Stack with PyTorch
+
+The release includes:
+ * Clear Linux* OS
+ * Runtimes (python)
+ * PyTorch with OpenBlAS
+
+## The Deep Learning Reference Stack with PyTorch and Intel® MKL
+
+The release includes:
+ * Clear Linux* OS
+ * Runtimes (python)
+ * PyTorch optimized using the Intel® Math Kernel Library
+ * PyTorch lightning
+ * Flair for Natural Language Processing
+
+## Deep Learning Compilers
+
+The release includes:
+ * TVM 0.6
+
+## How to get the Deep Learning Reference Stack
+
+The official Deep Learning Reference Stack Docker images are hosted at: https://hub.docker.com/u/clearlinux/.  Note that the Intel MKL-DNN-VNNI version is also referred to as the Intel® MKL with AVX-512 Deep Learning Boost in the documentation. 
+
+ * Pull from the [Eigen version](https://hub.docker.com/r/clearlinux/stacks-dlrs-oss/)
+ * Pull from the [Tensroflow 1.15 and Intel MKL-DNN version](https://hub.docker.com/r/clearlinux/stacks-dlrs-mkl/)
```

Comment:
Let's have the MKL versions first (1.15, 2.0 and then Eigen)

Reply:
+1